src/components/ActionControls/ActionControls.scss

```diff
@@ -15,9 +27,14 @@
         }
 
         .ba-annotation-input-container {
-            padding: 15px;
             width: 100%;
         }
+
+        .ba-annotation-label ~ .ba-action-controls .ba-action-controls-draw,
+        .ba-annotation-label ~ .ba-action-controls .ba-action-controls-highlight {
+            border-left: 0;
+            margin-left: 5px;
```

Comment:
is the `margin-left` still needed if it's already specified above on line 19?

Reply:
removed it